rock.osdeps

```diff
@@ -235,7 +235,7 @@ dc1394:
 
 vtk:
     debian,ubuntu:
-        '14.04,14.10,15.04,15.10':
+        '14.04,14.10,15.04,15.10,16.04':
```

Comment:
seems to work when changing the default python-vtk version to python-vtk6 in rock/rock.osdeps